### ISSUE TITLE
[CoE Starter Kit - BUG] Solution "Center of Excellence - Core Components" failed to import - 'AppModuleRoles' exception

### ISSUE THREAD

**Issue by zellos13** [microsoft/coe-starter-kit#8995]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Hi,

I am trying to import CoE Core solution but getting error after error. This time it's this one:

Solution "Center of Excellence - Core Components" failed to import: ImportAsHolding failed with exception :The uninstall operation will delete the base layer for the component 'AppModuleRoles' with id '7e755313-e33c-4390-8c15-8cd098896291'. The operation cannot continue because there are other managed layers over the base layer. You can use the solution layers to find out which other solutions are blocking the operation.

I've read this issue's solution [8245](https://github.com/microsoft/coe-starter-kit/issues/8245) but I was unable to resolve the problem. I don't really understand the explanation: "I was able to upgrade the solution after removing system administrator role which was assigned to user in the app share."

I removed all CoE Roles from people on that environment, so they are not used, and I unassigned roles from all apps:

![image](https://github.com/user-attachments/assets/e938bf4f-c613-49e1-bccc-384d8c9a9748)

![image](https://github.com/user-attachments/assets/182b6e0f-4dae-4cac-8383-b4669c80f59d)

Can you please suggest what should I do?

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.31

### What app or flow are you having the issue with?

Security Roles

### What method are you using to get inventory and telemetry?

None

**Comment by Felipe-Rafael-Rosa** [comment]
I think this will help you.
https://mega.co.nz/#!B31G2LaZ!uBBOCp9hLC7bq9kP8NC6s4DanQJJZuoFRr5FI_jkeic
I put the necessary dlls in the archive

**Comment by Jenefer-Monroe** [comment]
Hello. I'm afraid I'm not able to assist with this failure. It appears that the other user had to have product support assist them in finding the problematic connection between their custom MDAs and security roles. In short this appears to be a product bug and so I'm really not able to help find the cause.

**Comment by zellos13** [comment]
Hi,

How can I reach the product team? Should I create an MS support ticket from within the tenant, or is there any other way?

Kind regards,
Daniel

**Comment by Jenefer-Monroe** [comment]
Thats correct, a MS Support Ticket. Please let them know they can reach out to the CoE PM (Grant Archibald) if they have questions.

**Comment by zellos13** [comment]
Hello,

I created an MS Support ticket, and they told me that the Power Platform Admin View managed layer was causing the problem described in the issue.

This managed layer was created via the Customizations solution to the CoE - we don't know why it was a problem, but it solved an issue - removing the app from the customizations - unmanaged solution on DEV and importing it to the TEST environment without the app.

Then, we were able to import the solution.
